packages/vaex-ml/vaex/ml/__init__.py

```diff
@@ -224,6 +46,41 @@ def train_test_split(self, test_size=0.2, strings=True, virtual=True, verbose=Tr
         return train, test
 
 
+filename_spec = os.path.join(os.path.dirname(__file__), 'spec.json')
+
+if os.path.exists(filename_spec):
+    # add DataFrameAccessorML.<snake_name> wrapper methods
+    with open(filename_spec) as f:
+        try:
+            spec = json.load(f)
+        except json.decoder.JSONDecodeError:
+            pass  # we are generating the file probably
+        else:
+            for class_spec in spec:
+                def closure(class_spec=class_spec):
+                    def wrapper(self, features=None, transform=False, **kwargs):
```

Comment:
```suggestion
                    def wrapper(self, features=None, transform=True, **kwargs):
```
should we do this by default? if yes, I guess we need to change the tests.